src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -16,15 +16,30 @@ public Rectangle(Integer width, Integer length, String color) {
         this.color = color;
     }
 
-    public Integer getWidth() {
-        return width;
+    @Override
+    public boolean equals(Object object) {
+        if (this == object) {
+            return true;
+        }
+        if (object == null || getClass() != object.getClass()) {
+            return false;
+        }
+        Rectangle rectangle = (Rectangle) object;
+        return ((width == null && rectangle.width == null)
```

Comment:
```suggestion
        return (width == rectangle.width
```
just a note: this can be simplified